src/has-permissions-in-jwt-token.ts

```diff
@@ -0,0 +1,36 @@
+import { createDebug } from "./debug";
+import { AppPermission } from "./types";
+import { DashboardTokenPayload } from "./verify-jwt";
+
+const debug = createDebug("checkJwtPermissions");
+
+export const hasPermissionsInJwtToken = (
+  tokenData?: DashboardTokenPayload,
```

Comment:
```suggestion
  tokenData?: Pick<DashboardTokenPayload, 'user_permissions'>,
```

Interface Segregation -> in tests you don't need to provide unused field and this function should not require them if  its not using them